.github/workflows/ci.yaml

```diff
@@ -436,6 +439,118 @@ jobs:
           name: e2e-screenshots
           path: test/e2e/cypress/screenshots/
 
+  check-regression-label:
+    name: Check if the environment creation criteria are met, store in the job output
+    runs-on: ubuntu-latest
```

Comment:
Let's avoid `ubuntu-latest`. I know that it hardly affects here, but better pin it

Reply:
I'm with xabi on this, fixed version is extra save :+1: